test/scripts/build-image-transformer.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
This is a custom transformer image - I wouldn't give it such a generic name

Reply:
Updated, thanks.